pkgs/applications/science/electronics/linux-gpib/linux-gpib-user/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, stdenv
+, fetchurl
+, autoconf
+, libtool
+, bison
+, flex
+, automake
+}:
+
+stdenv.mkDerivation rec {
+  pname = "linux-gpib";
+  version = "4.3.6";
+
+  src = fetchurl {
+    url = "https://sourceforge.net/projects/linux-gpib/files/linux-gpib%20for%203.x.x%20and%202.6.x%20kernels/${version}/${pname}-${version}.tar.gz";
+    hash = "sha256-Gze4xrvkhEgn+J5Jhrycezjp2uhlD1v6aX0WGv4J2Jg=";
+  };
+
+  unpackPhase = ''
+    tar xf $src
```

Comment:
we can probably unpack the nested archive in postUnpack, too

Reply:
What do you mean by this? Can you give example?